hack/gen-python-sdk/post_gen.py

```diff
@@ -183,6 +184,29 @@ def update_python_sdk(src, dest, versions=("v1beta1")):
     # Clear working dictionary
     shutil.rmtree(src)
 
+def upgate_python_sdk_docs():
```

Comment:
update

Reply:
@shivaylamba I’ve removed the `update_python_sdk_docs()` method. It appears that Read the Docs updates the documentation automatically using the `.readthedocs.yaml` configuration, so there's no need for a separate workflow.